workspace/utilities/site-imports.xsl

```diff
@@ -5,6 +5,7 @@
 
 	<xsl:import href="com/site-header.xsl" />
 	<xsl:import href="com/site-header-logo-link.xsl" />
+	<xsl:import href="com/site-header-mobile-toggler.xsl" />
```

Comment:
Not sure if this should be included by default in the imports

Reply:
I agree, I removed it.